colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<p>Line 1: It's preferred to import in three blocks: standard library modules, then 3rd-party libraries, then your stuff (if any). So move <code>random</code>  to the top with <code>getpass</code>.</p><p>Lines 5-6: It's preferred to import in alphabetical order within each block.</p><p>Line 9: This is a repeat of Line 7.</p><p> </p><p>Lines 14-16: FYI, there's a bit more to do here to make Torch reproducible:</p><p> </p><pre class="ql-syntax" spellcheck="false">#&nbsp;Ensure&nbsp;deterministic&nbsp;behavior
torch.backends.cudnn.deterministic&nbsp;=&nbsp;True
random.seed(hash("setting&nbsp;random&nbsp;seeds")&nbsp;%&nbsp;2**32&nbsp;-&nbsp;1)
np.random.seed(hash("improves&nbsp;reproducibility")&nbsp;%&nbsp;2**32&nbsp;-&nbsp;1)
torch.manual_seed(hash("by&nbsp;removing&nbsp;stochasticity")&nbsp;%&nbsp;2**32&nbsp;-&nbsp;1)
torch.cuda.manual_seed_all(hash("so&nbsp;runs&nbsp;are&nbsp;repeatable")&nbsp;%&nbsp;2**32&nbsp;-&nbsp;1)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I didn't know about `torch.backends.cudnn.deterministic`. Thanks